Tests/MongoSwiftTests/CodecTests.swift

```diff
@@ -486,10 +463,10 @@ final class CodecTests: XCTestCase {
 
         let expected: Document = [
             "name": "Durian",
-            "points": 600.0,
```

Comment:
hm, this looks suspiciously like we don't support floating points anymore? 🤔

Reply:
so this is just with a decoder other than our own, and it's because of the behavior I described in the comment [here](https://github.com/mongodb/mongo-swift-driver/pull/77/files#diff-cd1c06a2751f196995f7a1679f3b0803R48) - basically, since 600.0 can be accurately represented as an `Int`, in the JSONDecoder ends up getting decoded as one regardless of whether the `.0` is there.

Reply:
Would it make sense to test an actual Double value that can't be represented as an integer in this case (e.g. 600.5), or is that not terribly important for this test?

Reply:
added a double value